people_tool/templates/people_tool/partials/courses.html

```diff
@@ -6,14 +6,46 @@ <h3 class="breadcrumbs">
     {% verbatim %}
     <small><i class="fa fa-chevron-right"></i></small>
     <a href="#/">Search People</a>
-    <small><i class="fa fa-chevron-right"></i></small>
-    {{ personId }}
+
+    <span ng-if="selectedPersonInfo.name_first">
```

Comment:
Could this fail in cases where the person has a blank first name? We might want a different if clause here and/or a fallback for the name_last, name_first text below (perhaps in a controller function or something).